addOns/database/database.gradle.kts

```diff
@@ -0,0 +1,48 @@
+import com.diffplug.spotless.extra.wtp.EclipseWtpFormatterStep
+import org.zaproxy.gradle.addon.AddOnStatus
+
+description = "Provides database engines and related infrastructure."
+
+val sqlite by configurations.creating
+configurations.api { extendsFrom(sqlite) }
+
+zapAddOn {
+    addOnName.set("Database")
+    addOnStatus.set(AddOnStatus.RELEASE)
```

Comment:
Was Release status intended or should it be bumped down to Alpha / Beta?

Reply:
It was, this is just providing a dependency. Agreed, better start in alpha as it will have move changes.

Reply:
Changed.